wsclient/src/test/java/org/hpccsystems/ws/client/DFUFileDetailWrapperTest.java

```diff
@@ -34,6 +34,12 @@
 @Category(org.hpccsystems.commons.annotations.BaseTests.class)
 public class DFUFileDetailWrapperTest
 {
+    private static final String SETOF = "RECORD\n" +
+            "STRING1 test;\n" +
+            "SET OF UTF8 field3set {DEFAULT ([''])};\n" +
+            "SET OF UTF8 field4set {DEFAULT (['test'])};\n" +
```

Comment:
Test case would benefit from i) empty set ii) more than one string in the list  iii) integer values in the list (for a set of integer).

Reply:
This is only a partial solution to cover the cases we need, specifically the DEFAULT (['']) case. It just happened to support the single value also.  This solution will not support more than one value in the list.  I agree the grammar should support any possible value DEFAULT can but... the grammar would need to be completely reworked to handle that we don't have anyone working in antlr currently.